.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -1024,6 +1111,18 @@ iam_AttachRolePolicy:
   synopsis: attach an &IAM; policy to a role.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.AttachRolePolicyExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.AttachPolicy
```

Comment:
These two examples are almost identical. Include only one--I'd pick the second because it's the tested one, but it's your call.

Reply:
This one still has two snippets.